src/Engine/System/RenderSystem.hpp

```diff
@@ -15,7 +15,9 @@
 #include "Shaders/DiffuseShader.hpp"
 #include "Shaders/BounderShader.hpp"
 
-
+#include <map>
+#include <string>
+#include <iostream>
 
```

Comment:
Please remove redundant #includes. 
string is included twice and iostream shouldn't need to be included in headers.

Reply:
We can remove map and string, but removing iostream gives me error: ‘cerr’ is not a member of ‘std’

Reply:
You right, iostream needs to stay included. I think same with map.
string is included twice tho